python-avd/pyavd/_eos_designs/structured_config/mlag/__init__.py

```diff
@@ -211,7 +214,7 @@ def ethernet_interfaces(self) -> list:
                 "speed": self.shared_utils.mlag_interfaces_speed,
             }
             if self.shared_utils.get_mlag_peer_fact("inband_ztp", required=False) is True:
-                ethernet_interface.update({"mode": "access", "vlans": self.shared_utils.get_mlag_peer_fact("inband_ztp_vlan")})
+                ethernet_interface.update({"switchport": {"mode": "access", "access_vlan": self.shared_utils.get_mlag_peer_fact("inband_ztp_vlan")}})
```

Comment:
The issue mentioned around inband ZTP is that we are missing `enabled: true` here. It can be solved now or later, but we need it in 5.0.

Reply:
Fixed